Mental illness/redme/readme.md

```diff
@@ -0,0 +1,42 @@
+# Mental Illness In Tech Industry
+
+## Tech’s quiet battle with depression
+
+The tech and startup world is built on the backs of incredibly bright minds. It's known for its innovation and resilience and a culture that fosters high-productivity.
+
+But it has a dark underbelly. 
+
+In this repository, you will know about mental illness and what people thinking about mental illness, Dose most people are aware of mental illness.
```

Comment:
This repository makes an effort to shed light on the mental health issues in Tech industry and how tech employees struggle through these illness. Furthermore, exploration on how Employers support and mitigate these situations at workplace has been done.